tests/puppeteer/.docker/prestashop/ps_v175

```diff
@@ -0,0 +1,42 @@
+FROM prestashop/base:5.6-apache
```

Comment:
Could we use https://hub.docker.com/r/prestashop/prestashop-git and make it configurable?

Reply:
and we will be able to do:
```bash
PS_VERSION=1.7.5.2 docker-compose -f docker-compose.autoupgrade.yml -f docker-compose.tests.yml up --build
```

Reply:
done